### PR TITLE
Update the Config API in 1.2.x Similar to Swan Lake

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the Config API in 1.2.x similar to Swan Lake.

Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/config-api/config_api.description [hunk 1]
```diff
@@ -1 +1,21 @@
...
+// * CLI parameters
+// * Environment variables
+// * Configuration files
+//
+// If a specific configuration defined in the file is also defined as an
+// environment variable, the environment variable takes precedence. Similarly,
+// if the same is set as a CLI parameter, it replaces the environment
+// variable value.
+//
+// The configuration file is in the TOML(v0.4) format.
```
Comment (pubudu91) [inline examples/config-api/config_api.description:14]: Remove this line. At least for now. Need to verify which version we support in 1.2.x

File: examples/config-api/config_api.metatags [hunk 2]
```diff
@@ -0,0 +1,2 @@
+description: The ballerina/config module in Ballerina provides an API to read configurations from various sources such as files, environment variables and CLI parameters.
```
Comment (pubudu91) [inline examples/config-api/config_api.metatags:1]: Did we add meta tags to BBEs in 1.2.x?

File: examples/config-api/config_api.description (hunk 1 above)
Comment (praneesha) [reply examples/config-api/config_api.description:14]: Removed.

File: examples/config-api/config_api.metatags (hunk 2 above)
Comment (praneesha) [reply examples/config-api/config_api.metatags:1]: Removed the `config_api.metatags` file.

**Review by pubudu91 — APPROVED** [review]

**Pull request merged**